### PR TITLE
fix: Upload Files button disappears for R/W users

### PR DESCRIPTION
**Pull request by SanketD92**
Issue #, if available:
GALI-606

Description of changes:
Upload Files button the studies page would not appear for users who are study admins AND possess R/W or R/O permission. This fix ensures study admins and R/W permitted users have access to the Upload Files button.

Checklist: 

* [x] Have you successfully deployed to an AWS account with your changes? 
* [x] Have you linted your code locally prior to submission?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran unit tests and manual tests with your changes locally?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: main/solution/ui/src/models/hello/HelloStore.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ import { Hello } from './Hello';
 // ==================================================================
 const HelloStore = BaseStore.named('HelloStore')
   .props({
-    hellos: types.array(Hello),
+    hellos: types.optional(types.array(Hello), []),
```
Comment (jn1119) [inline main/solution/ui/src/models/hello/HelloStore.js:28]: nit: how is this file related to the change?

Comment (SanketD92) [reply main/solution/ui/src/models/hello/HelloStore.js:28]: Not related to the fix, but a week ago the lobx-state-tree version was updated which caused local debugging to fail. This is the fix for that.

**Review by jn1119 — APPROVED** [review]
